api/urls.py

```diff
@@ -172,6 +173,7 @@
     ),
     path("importPurchases/", PurchasesImportView.as_view(), name="purchases_import"),
     path("importCanteens/", CanteensImportView.as_view(), name="canteens_import"),
+    path("importDiagnostics/simple/", DiagnosticsSimpleImportView.as_view(), name="import_diagnostics_simple"),
```

Comment:
```suggestion
    path("importDiagnostics/simple/", DiagnosticsSimpleImportView.as_view(), name="diagnostics_simple_import"),
```
pour homogénéiser avec les autres imports (suite à #6052)